docs/admin/index.rst

```diff
@@ -122,6 +122,41 @@ For instance, make a user admin on GitLab.
 3. modify any users you want to modify (e.g. to make them admin)
 4. turn the automatic redirect back on
 
+Enabling notebook for anonymous users
+-------------------------------------
+
+Interactive sessions launched from public notebooks can be made available to logged-out users. However,
+this feature is deactivated by default. In order to activate it, you need a secondary deployment of the
+renku-notebooks chart (including a secondary Jupyterhub as a dependency) to handle the anonymous
+("temporary") sessions.
+
+1. Set :code:`.Values.global.anonymousSessions.enabled: true` in your values.yaml file.
+2. Deploy renkulab (or update your existing deployment) just as you would otherwise.
+3. Create an additional kubernetes namespace:
```

Comment:
can't this be automatically done? You could configure the name of the namespace in the values file with some sensible default and provide a template that gets rendered only if the global toggle switch is on.